canonical-web-and-design/maas-ui
# refactor(ui): config slice

## Done

- Config config actions and reducers to a slice.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to Settings -> General and change something like the MAAS name or notification settings.
- Refresh and the data should have been saved.

## Fixes

Fixes: #1404.
